src/certificate.rs

```diff
@@ -223,16 +441,30 @@ impl Certificate {
             _ => return Err(Error::InvalidObject),
         };
 
+        let serial = parsed_cert.tbs_certificate.serial;
+        let issuer = format!("{}", parsed_cert.tbs_certificate.issuer);
```

Comment:
Seems like a roundabout way of calling `to_string()`?

Reply:
lol, good point :laughing: